src/prefect/cli/__init__.py

```diff
@@ -22,3 +22,7 @@
 # Only load workers CLI if enabled via a setting
 if prefect.settings.PREFECT_EXPERIMENTAL_ENABLE_WORKERS.value():
     import prefect.experimental.cli.worker
+
+# Only load artifacts CLI if enabled via a setting
+if prefect.settings.PREFECT_EXPERIMENTAL_ENABLE_ARTIFACTS.value():
```

Comment:
I know I'm the one that added the statement above this one, but I think the preferred way to check if an experiment is enabled is with the `experiment_enabled` function.
```suggestion
if experiment_enabled("artifacts"):
```